packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -123,17 +132,17 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
     protected createInputTypes(model: ModelDefinition) {
         const modelFields = Object.values(model.graphqlType.getFields());
         const inputName = getInputTypeName(model.graphqlType.name);
-        
-        //TODO relationships?
+
         return new GraphQLInputObjectType({
             name: inputName,
-            fields: () => (modelFields.filter((field: GraphQLField<any, any>) => {
-                const fieldBaseType = getBaseType(field.type);
-
-                return !isObjectType(fieldBaseType);
-            }).reduce((fieldObj: any, current: any) => {
-                const fieldType = current.type;
-                fieldObj[current.name] = { type: getNullableType(fieldType), description: '' };
+            fields: () => (modelFields.filter(canInputField).map((field: GraphQLField<any, any>) => {
```

Comment:
I found it hard to get this. `canInputField` is not a very descriptive filter.
Offix plugin rebuilds input types (and there will be multiple plugins getting it.
It could be issue to improve the way that we build this and in this PR let's improve naming in core